cdap-ui/app/features/dashboard/templates/staticdashboard.html

```diff
@@ -1,5 +1,5 @@
 <div class="row" ng-controller="WidgetColCtrl">
-  <div ng-class="col-xs-12 col-md-6"
+  <div ng-class="{'col-xs-12 col-md-6' : true}"
```

Comment:
@nitinmotgi Is this changed? if we want 3 columns in the system dashboards then ```class="col-xs-12 col-md-4"``` is what we needed.. I see ```widgetsPerRow``` in controller to be 3. Just verifying..